src/styles/typography.css

```
@@ -0,0 +1,87 @@
+  --text-body2: 18px;
+  --text-label: 14px;
+  --text-caption: 12px;
+
+  --font-weight-bold: 700;
+  --font-weight-semibold: 600;
+  --font-weight-medium: 500;
+  --font-weight-regular: 400;
+}
+
+@theme {
+  /*font family*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)